jira.go

```diff
@@ -219,16 +219,16 @@ func (j *Jira) IssuesAssignedTo(user string, maxResults int, startAt int) IssueL
 }
 
 // search an issue by its id
-func (j *Jira) Issue(id string) Issue {
+func (j *Jira) Issue(id string) (*Issue, error) {
```

Comment:
We cannot change the method signature, also from the code Issue is not pointer but a direct structure ...